### PR TITLE
OSDOCS-5281: update module to procedure format

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5281

Link to docs preview:
https://55774--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-troubleshooting.html#microshift-troubleshooting-nodeport_microshift-known-issues

QE review:
N/A this a solely a docs-format fix.

Additional information:

### CODE REVIEW

**Review by jeana-redhat — COMMENTED** [review]
Went over this with Alberto, just some small stuff/questions - thanks for a good example PR :D

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/microshift-troubleshooting-nodeport.adoc [hunk 1]
```diff
@@ -7,27 +7,31 @@
...
 $ iptables-save | grep NODEPORT
 ----
++
 .Example output
 [source, terminal]
 ----
 -A OUTPUT -j OVN-KUBE-NODEPORT
 -A OVN-KUBE-NODEPORT -p tcp -m addrtype --dst-type LOCAL -m tcp --dport 30326 -j DNAT --to-destination 10.43.95.170:80
 ----
+OVN-Kubernetes configures the OVN-KUBE-NODEPORT chain in iptable NAT table to match the destination port and DNATs the packet to the backend clusterIP service. The DNATed packet is then routed to the OVN network through gateway bridge br-ex via routing rules on the host.
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-troubleshooting-nodeport.adoc:26]: Ya, a little bit. Cleaned up.
It's just "iptables" in the OCP docs as far as I can tell. 
Ah, but OpenFlow is a branded protocol with caps, I just learned.

[the remark this answers is not in the document]
Comment (jeana-redhat) [reply modules/microshift-troubleshooting-nodeport.adoc]: neat - read over your changes so I can get more familiar with this terminology. Thanks :)

**Pull request merged**